src/main/java/org/folio/spring/service/TenantService.java

```diff
@@ -0,0 +1,58 @@
+package org.folio.spring.service;
+
+import static org.apache.commons.lang3.BooleanUtils.isTrue;
+
+import java.sql.ResultSet;
+import liquibase.exception.LiquibaseException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.spring.FolioExecutionContext;
+import org.folio.spring.exception.NotFoundException;
+import org.folio.spring.liquibase.FolioSpringLiquibase;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Service;
+
+@Log4j2
+@RequiredArgsConstructor
+@Service
+public class TenantService {
+  private static final String DESTROY_SQL = "DROP SCHEMA IF EXISTS %1$s CASCADE; DROP ROLE IF EXISTS %1$s";
+  private static final String EXIST_SQL = "SELECT EXISTS(SELECT 1 FROM pg_namespace WHERE nspname=?)";
+
+  private final JdbcTemplate jdbcTemplate;
+  private final FolioExecutionContext context;
+  private final FolioSpringLiquibase folioSpringLiquibase;
+
+  public void createTenant(String tenant) throws LiquibaseException {
+    if (folioSpringLiquibase != null) {
+      var tenantId = context.getTenantId();
```

Comment:
I assume that parameter `tenant` and variable `tenantId` are the same value. Use one of them, please.